### ISSUE TITLE
Dex percent decimal too low

### ISSUE THREAD

**Issue by Steedr** [Neovitalism/PixelmonExtension#1]
If you parse `%pixelmon_trainer_dex_percent_caught%` it returns 0.08839xxx
It should be 8.839xxx
Same for
```
%pixelmon_trainer_dex_percent_caught_rounded% (returns 0.09)
%pixelmon_trainer_dex_percent_caught_int% (returns 0)
%pixelmon_trainer_dex_percent_caught_gen_[1-8]%
%pixelmon_trainer_dex_percent_caught_gen_[1-8]_rounded%
%pixelmon_trainer_dex_percent_caught_gen_[1-8]_int%
%pixelmon_trainer_dex_percent_seen%
%pixelmon_trainer_dex_percent_seen_rounded%
%pixelmon_trainer_dex_percent_seen_int%
%pixelmon_trainer_dex_percent_seen_gen_[1-8]%
%pixelmon_trainer_dex_percent_seen_gen_[1-8]_rounded%
%pixelmon_trainer_dex_percent_seen_gen_[1-8]_int%
```

And one small thing, just noticed you have placeholder for held item in code but not on the wiki

**Comment by Neovitalism** [comment]
Will fix when I get home later, appreciate - and oops on the held item. The wiki was quite a doosy to write lol - that I believe I can fix now.

Does this also apply to stuff like IV_percent?

**Comment by Steedr** [comment]
No, IVs and EVs percentages are fine

**Comment by Neovitalism** [comment]
Appreciate, will fix as soon as I get the chance

**Comment by Steedr** [comment]
Oh, just found out total EVs isn't correct, you have a 520 instead of 510

**Comment by Neovitalism** [comment]
All fixed in [PixelmonExtension 1.0.1](https://github.com/Neovitalism/PixelmonExtension/releases/tag/1.0.1)
Thank you <3
